rollup/da_syncer/da_syncer.go

```diff
@@ -0,0 +1,53 @@
+package da_syncer
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/core"
+	"github.com/scroll-tech/go-ethereum/core/types"
+	"github.com/scroll-tech/go-ethereum/log"
+	"github.com/scroll-tech/go-ethereum/trie"
+)
+
+type DaSyncer struct {
```

Comment:
```suggestion
type DASyncer struct {
```

I'm not sure if `DASyncer` is a fitting name. First I thought it would sync something but it essentially derives the state from the DA data and executes the next block iiuc no?